namerd/storage/zk/src/main/scala/io/buoyant/namerd/storage/experimental/ZkDtabStoreInitializer.scala

```diff
@@ -21,6 +23,20 @@ case class zk(
   override def mkDtabStore: DtabStore = new ZkDtabStore(
     hosts.mkString(","),
     pathPrefix.getOrElse("/dtabs"),
-    sessionTimeout
+    sessionTimeout,
+    authInfo,
+    acls.getOrElse(Seq(Acl.AnyoneUnsafe))
   )
 }
+
+case class AuthInfo(scheme: String, auth: String)
+
+case class Acl(scheme: String, id: String, perms: String) {
+  if (!Acl.validPerms(perms))
+    throw new IllegalArgumentException(s"$perms is not a valid permission string")
+}
+
+object Acl {
+  val validPerms = "[crwda]+".r.pattern.asPredicate.test _
```

Comment:
nit: this allows strings with duplicates that I assume are invalid, like "ccccc".